### ISSUE TITLE
Store Sections and Pages as XML

### ISSUE THREAD

**Issue by kanduvisla** [symphonycms/symphony-2#1002]
I believe this was already done by the Symphony 3 project, but I was wondering the other day that it would be better to store sections and pages in configuration-XML files, rather than directly in the database.

The reason for this is interchangability. When creating websites as a full-time job, it's rather a waste of time to have to recreate the same logic for news-sections, faq, images gallery, etc. over and over again. It would be great to:
- store the structure of sessions in seperate XML-files (just like the scaffold-extension sort of does with JSON-files, but then only for importing and exporting)
- store pages configuration in an XML file (type, datasources, events, etc.)

This change could provide for interchangable pieces of code. Think about a couple of files which you could throw in your Symphony root, and they would provide the sections, (demo?)pages, utililities, (demo?)css and javascript for fully functional widgets for your site. This could be a great addition next to extensions and XSLT-utilities.

Now, before we start with any further details on why this (or why not) this could work, please share your thoughts on something like this. I'm sure I'm not the only one creating news sections, faqs and image slideshows over-and-over again.
